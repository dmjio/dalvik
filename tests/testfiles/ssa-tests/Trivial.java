class Trivial {
  static void entry() {}
}
