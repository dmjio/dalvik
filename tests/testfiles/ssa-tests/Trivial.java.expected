SimplePhi { numBlocks = 1
          , phis = []
          , blockPhis = []
          , assignedPhis = []
          }